### ISSUE TITLE
Default watermark contrast ratio is not accessible

### ISSUE THREAD

**Issue by shab5** [xyflow/xyflow#4702]
### What platform were you using when you found the bug?

- React Flow / Svelte Flow version: React Flow 12.2
- Browser and version: Chrome v129.0.6668.70
- OS and version: Linux

### Describe the Bug

The contrast ratio of the default "React Flow" watermark is not sufficient for those with impaired vision.

### Steps to reproduce the bug or issue

1. Load a page with the <ReactFlow /> component.
2. The watermark appears in the lower right corner.

### Expected behavior

I expect the watermark to have sufficient contrast ratio for those with impaired vision.

**Comment by peterkogo** [comment]
The attribution serves no function for your application and is designed to be visually unintrusive. This is obviously against accessible design guidelines, but I don't think there is a middle ground here. You can, however, edit the styles of `.react-flow__attribution` to increase the contrast.
